cmake/Modules/FindTensorflow.cmake

```diff
@@ -25,6 +25,11 @@ if (LEN EQUAL "4")
     if (Tensorflow_VERSION VERSION_GREATER_EQUAL "2.6")
         # XLA implementations and helpers for resource variables are in _pywrap_tensorflow_internal.so
         set(Tensorflow_LIBRARIES "${Tensorflow_LIBRARIES} ${Tensorflow_LIB_PATH}/python/_pywrap_tensorflow_internal.so")
+        set(lib_tensorflow_cc_so_path "${Tensorflow_LIB_PATH}/libtensorflow_cc.so.2")
```

Comment:
what does the `.2` stand for?

Reply:
I hope that stands for TensorFlow 2.x, but I have not dived into the build system to confirm. The main library in that path is `libtensorflow_framework.so.2`. We get the latter via `tf.sysconfig.get_link_flags()`, but I couldn't find an API to resolve the new `libtensorflow_cc.so.2`.